evm/src/executor/backend/mod.rs

```diff
@@ -588,15 +593,16 @@ impl Backend {
         false
     }
 
-    /// In addition to the `_failed` variable, `DSTest::fail()` stores a failure
-    /// in "failed"
-    /// See <https://github.com/dapphub/ds-test/blob/9310e879db8ba3ea6d5c6489a579118fd264a3f5/src/test.sol#L66-L72>
-    pub fn is_global_failure(&self) -> bool {
-        let index = U256::from(&b"failed"[..]);
-        self.storage(h176_to_b176(CHEATCODE_ADDRESS), u256_to_ru256(index))
-            .map(|value| value == revm::primitives::U256::from(1))
-            .unwrap_or_default()
-    }
+    // todo:error2215 dead code
```

Comment:
I've checked and seems like this function is not used anywhere, do we need it?

Reply:
If we don't use it, I don't think we need it.